Address some style nits [NFCI]

vtjnash opened the pull request:
Preparatory work for https://github.com/libuv/libuv/pull/2953

CI: https://ci.nodejs.org/job/libuv-test-commit/2002/

vtjnash commented:
Hm, seems like IBMI is failing at the harder pipe_set_non_blocking test, but I'm not sure how to investigate.

richardlau commented:
> Hm, seems like IBMI is failing at the harder pipe_set_non_blocking test, but I'm not sure how to investigate.

Ping @libuv/ibmi

bnoordhuis commented:
@vtjnash In the interest of progress, I suggest rolling back the change to `test/test-pipe-set-non-blocking.c` and opening a separate pull request for that.

vtjnash commented:
Yeah, I dropped the extra timeout for now, but kept the adjustment to use libuv's fs interface instead of posix. I also found a few other files that were missing copyright notice (`grep -RIL Copyright src/ test | grep '\.[ch]$'`) and added those here too.

CI: https://ci.nodejs.org/job/libuv-test-commit/2011/
